code/modules/antagonists/abductor/abductee/abductee_objectives.dm

```diff
@@ -1,54 +1,37 @@
 /datum/objective/abductee
 	completed = 1
+	explanation_text = "<b>They</b> took you... you can't remember...? The pale faces.. the hands... <b>They</b> control us now. You must protect yourself."
 
-/datum/objective/abductee/random
+/datum/objective/abductee/fearful
+	explanation_text = "They're in league with <b>them</b>. You know their plans now. You must tell the others:<br>"
 
-/datum/objective/abductee/random/New()
-	explanation_text = pick(world.file2list("strings/abductee_objectives.txt"))
+/datum/objective/abductee/fearful/New()
+	var/target = pick(list("Security","Medical","Command"))
+	var/transgression = pick(list("putting drugs in the water supply","putting nanomachine trackers into our bodies","using the cameras to track our emotions","putting drugs in the food"))
+	explanation_text += "[target] is [transgression] to keep us compliant and sheepish. They know that you know it. Hide! Run! Don't let them catch you!"
 
-/datum/objective/abductee/steal
-	explanation_text = "Steal all"
 
-/datum/objective/abductee/steal/New()
-	var/target = pick(list("pets","lights","monkeys","fruits","shoes","bars of soap", "weapons", "computers", "organs"))
-	explanation_text+=" [target]."
+/datum/objective/abductee/violent
+	explanation_text = "You are filled with an otherworldly rage simmering within. Threatening to boil over at the slightest provocation:<br>"
 
-/datum/objective/abductee/paint
-	explanation_text = "The station is hideous. You must color it all"
-
-/datum/objective/abductee/paint/New()
-	var/color = pick(list("red", "blue", "green", "yellow", "orange", "purple", "black", "in rainbows", "in blood"))
-	explanation_text+= " [color]!"
-
-/datum/objective/abductee/speech
-	explanation_text = "Your brain is broken... you can only communicate in"
+/datum/objective/abductee/violent/New()
+	// we use suit sensors because we really want to make sure that this person can be found by our little crackhead here
+	var/target
+	if(GLOB.suit_sensors_list)
+		target = "[pick(GLOB.suit_sensors_list.len > 2)]"
+	else
+		target = "the first person I see"
 
-/datum/objective/abductee/speech/New()
-	var/style = pick(list("pantomime", "rhyme", "haiku", "extended metaphors", "riddles", "extremely literal terms", "sound effects", "military jargon", "three word sentences"))
-	explanation_text+= " [style]."
+	var/list/your_poison = list("I am [target], <b>they</b> have changed my face and tried to replace me by sending a clone!! I must kill them first and resume my life as my original identity.","No one believes me, no one loves me. I will show them how badly they will miss me, I will show them ALL! I must KILL MYSELF as violently and publicly as possible.","THEY'RE ALL INSANE!!! THEY ALL WANT ME DEAD!!! I MUST KILL THEM ALL I MUST SHOW THEM THAT I WON'T GO DOWN SO EASY!!! TRY ME!!!")
```

Comment:
Remove the objective that forces people to kill themselves. If a replacement is required use something like form a violent public execution.